### ISSUE TITLE
valid_dice_Miscellaneous=0

### ISSUE THREAD

**Issue by haiqinzhong** [vqdang/hover_net#68]
Hi, @simongraham and @vqdang    
     I'm sorry to bother you again. When I randomly get my valid set from the training set,  and the number of image in vaild set is same as test set, I get the **valid_dice_Miscellaneous=0** in training after 50 epochs. Sometimes it appears at 70th epoch, sometimes it appears at 74th epoch. And after that epoch, the value of valid_dice_Miscellaneous is 0. And I wonder know why.

**Comment by haiqinzhong** [comment]
Sorry, can you tell me why? Or when train the model, it is better when the value of valid_dice_Miscellaneous close to 0 or close to 1?

**Comment by caprioGirl** [comment]
hi @haiqinzhong , did you find an answer to this? as I am getting the same issue
